### PR TITLE
CIV-13115 Dashboard Notification - Defendant Full admit pay immediately (Claimant)

### PR DESCRIPTION
**Pull request by kenneth-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-13115

### Change description ###
Set up dashboard notification for the Claimant when Defendant choose Full Admit and Pay Immediately

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by ElifDenizEsea — APPROVED** [review]

**Review by ElifDenizEsea — APPROVED** [review]

File: src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/dashboard/scenarios/claimant/CCJRequestedScenarioTest.java [hunk 1]
```diff
@@ -30,7 +30,6 @@ void should_create_ccj_requested_scenario() throws Exception {
             .toBuilder()
             .legacyCaseReference("reference")
             .ccdCaseReference(Long.valueOf(caseId))
-            .respondent1ResponseDeadline(responseDeadline.atStartOfDay())
```
Comment (neeta-hmcts) [inline src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/dashboard/scenarios/claimant/CCJRequestedScenarioTest.java:33]: why was this removed?

Comment (kenneth-hmcts) [reply src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/dashboard/scenarios/claimant/CCJRequestedScenarioTest.java]: Added back

Comment (portizdev) [reply src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/dashboard/scenarios/claimant/CCJRequestedScenarioTest.java]: This comes from Max PR, I think we should check it with him @m-meulendijks-v1

Comment (kenneth-hmcts) [reply src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/dashboard/scenarios/claimant/CCJRequestedScenarioTest.java]: According to Max, it wasn't relevant to the test, so it is removed.
@neeta-hmcts @portizdev

**Review by dharmendrak — APPROVED** [review]
